frontend/src/pages/modelCatalog/ModelCatalogRoutes.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import ModelCatalogCoreLoader from './ModelCatalogCoreLoader';
+import ModelDetailsPage from './ModelDetailsPage';
+
+const ModelCatalogRoutes: React.FC = () => (
+  <Routes>
+    <Route path={'/:modelCatalog?/*'} element={<ModelCatalogCoreLoader />} />
+    <Route path="tempDetails" element={<ModelDetailsPage />} />
```

Comment:
Hmm, this should be a child of the ModelCatalogCoreLoader route right? like we have for the subroutes of model registry, so we can eventually have the ModelCatalogContextProvider in ModelCatalogCoreLoader. I won't block this PR on it so we can keep moving though. I guess I did specify /tempDetails as a top level route in the story, that's on me.

Reply:
Oh, I missed this. I think this can be fixed as part of this ticket: https://issues.redhat.com/browse/RHOAIENG-18957

Reply:
@manaswinidas when you do that story we also may want to rename the `:modelCatalog?` route param something like `:modelCatalogSource?` or `:source?` because it will be the name of a `ModelCatalogSource` object we'll match